---
exp_x_distance: 1.5
x_max_distance: 1.55
x_min_distance: 1.45
y_max_distance: 0.05
max_angle: 0.15
max_step: 200
pos_noise: 0.01
obs_with_velocity: True
obs_with_time: False
noise: False
time_noise: 0.05
low_bound:
  - -0.3
  - -0.4
  - -0.15
robot2_low_bound:
  - -0.2
  - -0.2
  - -0.1
high_bound:
  - 0.3
  - 0.4
  - 0.15
robot2_high_bound:
  - 0.2
  - 0.2
  - 0.1
...